README.md

```diff
@@ -1,2 +1,374 @@
+
+
+# ✅ Good
+list = [
+  "value1",
+  "value2",
+  "value3",
+  "value4",
+]
+
+# ✅ Good
+list = ["value1", "value2", "value3"]
+```
+
+**This is also valid for Maps and Keyword Lists.**
+
+[Back to top ⬆️](#table-of-contents)
+
+## Blank Lines
+
+Blank lines are important in some cases, but in some others can make the code more unreadable.
+The examples below will give more details about this.
+
+#### Module
+
+Modules should not contain a blank line between the `moduledoc` and the module name.
+
+```elixir
+# ❌ Bad
+defmodule Jungsoft.NicePlaceToWork do
+
+  @moduledoc false
+
+  # DO SOMETHING
+end
+
+
+# ✅ Good
+defmodule Jungsoft.NicePlaceToWork do
+  @moduledoc false
+
+  # DO SOMETHING
+end
+```
+
+The module `end` should not contain a blank space either.
+
+```elixir
+# ❌ Bad
+defmodule Jungsoft.BestPeople do
+  @moduledoc false
+
+  def foo do
+    true
+  end
+
+end
+
+
+# ✅ Good
+defmodule Jungsoft.BestPeople do
+  @moduledoc false
+
+  def foo do
+    true
+  end
+end
+```
+
+#### Function
+
+Different functions should contain exactly one blank space between each other.
+
+```elixir
+# ❌ Bad
+defmodule Jungsoft.GoodCoffee do
+  @moduledoc false
+
+  def foo, do: true
+
+
+  def bar, do: true
+  def func, do: true
+end
+
+
+# ✅ Good
+defmodule Jungsoft.GoodCoffee do
+  @moduledoc false
+
+  def foo, do: true
+
+  def bar, do: true
+
+  def func, do: true
+end
+```
+
+Functions should not contain a blank line between the function call and the parameters. In addition, they should not end with a blank line.
+
+```elixir
+# ❌ Bad
+defmodule Jungsoft.BeautifulOfficeView do
+  @moduledoc false
+
+  def foo do
+
+    1 + 1
+
+  end
+end
+
+
+# ✅ Good
+defmodule Jungsoft.BeautifulOfficeView do
+  @moduledoc false
+
+  def foo do
+    1 + 1
+  end
+end
+```
+
+#### With
+
+The `with do` parameters should not be separated by blank spaces. Except when `else` contains multiple options. In this case should use the same logic as `case` and `cond` normal style (see [here](https://github.com/christopheradams/elixir_style_guide#multiline-case-clauses)).
+
+```elixir
+# ❌ Bad
+with  %StarWars{} <- Jungsoft.function_1(),
+      true <- Jungsoft.function_2()
+do
+
+  ## DO SOMETHING
+
+else
+
+  %StarTrek{} ->
+    {:error, "Wrong choice"}
+
+  false ->
+    ## DO SOMETHING
+
+end
+
+
+# ✅ Good
+with  %StarWars{} <- Jungsoft.function_1(),
+      true <- Jungsoft.function_2()
+do
+  ## DO SOMETHING
+else
+  %StarTrek{} ->
+    {:error, "Wrong choice"}
+
+  false ->
+    ## DO SOMETHING
+end
+```
+
+[Back to top ⬆️](#table-of-contents)
+
+## Dangling Commas
+
+Objects with the possibility to span in multiple lines should contain a comma dangle in the last parameter. See an alias example below.
+
+```elixir
+# ❌ Bad
+alias Jungsoft.{
+  Module1,
+  Module2
+}
+
+
+# ✅ Good
+alias Jungsoft.{
+  Module1,
+  Module2,
+}
+```
+
+**This is also valid for Maps, Lists and Keyword Lists.**
+
+[Back to top ⬆️](#table-of-contents)
+
+# Schemas
```

Comment:
Since these are specific to libraries (ecto and absinthe), I think this section should be the last one, giving more focus to the general elixir language guidelines.